### ISSUE TITLE
Remove dishwasher warning from OS-HPXML response

### ISSUE THREAD

**Issue by jefffriesen** [RadiantLabs/modeling-api-docs#14]
Current, if there is any legitimate error, the modeling engine will pass back warnings (which can generally be ignored). If there are no true errors, warnings are not passed back. So we get a situation like this:

```
"errors": [
        {
            "message": "[\"Warning: No dishwasher specified, the model will not include dishwasher energy use.\" \"Error: /tmp/modeling-api/dir2/2023091216415567346744/13992727_BASE.xml: Expected CompressorType to be 'single stage' or 'two stage' or 'variable speed' [context: /HPXML/Building/BuildingDetails/Systems/HVAC/HVACPlant/CoolingSystem]\" \"Error: /tmp/modeling-api/dir2/2023091216415567346744/13992727_BASE.xml: Expected CompressorType to be 'single stage' or 'two stage' or 'variable speed' [context: /HPXML/Building/BuildingDetails/Systems/HVAC/HVACPlant/CoolingSystem[CoolingSystemType=\\\"central air conditioner\\\"]]\"]",
            "code": "openstudio-creating-input-unsuccessful"
        }
    ]
```
Our defaulting engine may include a dishwasher or not, depending on regional characteristics, which is desired behavior. If the API user wants to specify a dishwasher they can, otherwise they can leave it up to us. The real error is compressorType. 

We will try to remove the dishwasher and other irrelevant warnings from error messages.

**Comment by lukeduran** [comment]
Hi @jefffriesen, can warnings go into a warnings array so that we can parse through both errors and warnings accordingly?

**Comment by jefffriesen** [comment]
@lukeduran looking into this now

**Comment by jefffriesen** [comment]
@lukeduran I agree a warning array would be better and are now planning on doing that. A few points:
1. Even if we do have separate warnings, the dishwasher warning is annoying and I want to remove it. It's not a problem if you don't have a dishwasher in the house (I don't). If you want to explicitly set a dishwasher that's fine. It's just like reporting back a building characteristic, which shouldn't be a warning. So again, I'm thinking that we remove it but pass through other more helpful warnings.
2. This seems lower priority than other issues, such as automated measures. In fact, other than the dishwasher warning, you likely won't get any warnings returned in the near future. Is that what you've seen?
